lib/mapping.ts

```diff
@@ -21,6 +22,42 @@ export default class Mapping implements IMapping {
     return this.mapper.map(stringOrArray);
   }
 
+  public or(source: string) {
+
+    if (!source) {
+      throw new Error("'source' is required for the 'or' method");
+    }
+
+    if (Array.isArray(source)) {
+      throw new Error("The 'or' method can only be used with single selections");
+    }
+
+    if (Array.isArray(this.source) && this.orMode === false) {
```

Comment:
I am a bit confused... If `Array.isArray(this.source)` is true you'll never get here because the previous condition in line https://github.com/midknight41/map-factory/pull/27/files#diff-b744beb2e7ab971c40e5f3a712844646R31 is going to throw an error